src/test/java/com/wikia/webdriver/testcases/mercurytests/VerbatimTests.java

```diff
@@ -0,0 +1,43 @@
+package com.wikia.webdriver.testcases.mercurytests;
+
+import com.wikia.webdriver.common.contentpatterns.MercurySubpages;
+import com.wikia.webdriver.common.core.annotations.Execute;
+import com.wikia.webdriver.common.core.api.ArticleContent;
+import com.wikia.webdriver.common.core.configuration.Configuration;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.mercury.LoginPage;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.verbatim.widgets.TwitterPageObject;
+
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.util.concurrent.TimeUnit;
+
+/**
+ * @ownership: Content X-Wing
+ */
+// Uncoment after finish all verbatim tags
+//@Test(groups = {"MercuryVerbatimTests", "Mercury"})
+public class VerbatimTests extends NewTestTemplate {
```

Comment:
verbatim is just a name of tag in mediawiki. The tested functionality is a widget. 
change the class name to  widgetTests or thirdparties